### PR TITLE
Fixes being-sent message handling in chat

### PR DESCRIPTION
**Pull request by ottpeter**

### CODE REVIEW

File: src/components/Messages/Message/Message.tsx [hunk 1]
```diff
@@ -101,6 +101,7 @@ const Message: React.FC<MessageProps> = ({
           <span>{likeCount ? likeCount : ""}</span>
           <button
             className="message__right-side__message-controls_like"
+            disabled={data.threadId === "being-sent"}
```
Comment (bosi95) [inline src/components/Messages/Message/Message.tsx:104]: use a constant for threadId being-sent, it appears in multiple lines

File: src/components/Messages/Messages.tsx [hunk 2]
```diff
@@ -56,7 +56,7 @@ const Messages: React.FC<MessagesProps> = ({
   };
   
   // Schedule a scroll after the state update if we're already at the bottom
-  if (isScrolledToBottom()) {
+  if (true || isScrolledToBottom()) {
```
Comment (bosi95) [inline src/components/Messages/Messages.tsx:59]: why true is needed here ?

**Review by bosi95 — APPROVED** [review]
+

File: src/components/Messages/Messages.tsx (hunk 2 above)
Comment (ottpeter) [reply src/components/Messages/Messages.tsx:59]: I think I just turned off the if-branch for testing. Scroll will need to be fixed, this will be changed.

**Review by rolandlor — COMMENTED** [review]
On my side when I try to send a message the message is visible in a transparent way during connecting to chat, and after the message is not visible.

**Review by rolandlor — APPROVED** [review]

**Pull request merged**